src/EditorFeatures/CSharp/SplitStringLiteral/SplitStringLiteralCommandHandler.StringSplitter.cs

```diff
@@ -150,7 +149,7 @@ private string GetIndentString(SyntaxNode newRoot)
             {
                 var newDocument = Document.WithSyntaxRoot(newRoot);
 
-                var indentationService = (IBlankLineIndentationService)newDocument.GetLanguageService<ISynchronousIndentationService>();
+                var indentationService = newDocument.GetLanguageService<Indentation.IIndentationService>();
```

Comment:
this is from the https://github.com/dotnet/roslyn/pull/33978 PR.

Reply:
why not just use IIndentationService?